spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/ldap/embedded/EmbeddedLdapProperties.java

```diff
@@ -109,4 +123,34 @@ public void setPassword(String password) {
 
 	}
 
+	public static class Validation {
+
+		/**
+		 * Enable LDAP schema validation
+		 */
+		private boolean enabled = true;
```

Comment:
Those really are two features. Most of the time you'll only need to disable the validation altogether and thats basically what the flag is for. 

But if you don't have custom attribute types or object classes you can still have validation against the default standard schema provided by unboundid as it is initialized in `InMemoryDirectoryServerConfig` with `Schema.getDefaultStandardSchema()`. So not providing a custom schema here doesn't mean you don't want validation.

Of course it doesn't make sense to set a schema and disable validation...